src/System.Private.CoreLib/shared/System/Threading/Tasks/Future.cs

```diff
@@ -377,8 +378,8 @@ private string DebuggerDisplayMethodDescription
         {
             get
             {
-                Delegate d = m_action;
-                return d != null ? d.Method.ToString() : "{null}";
+                Delegate? d = m_action;
```

Comment:
Couldn't this be simplified? -> `return m_action?.Method.ToString() ?? "{null}";` ?

Reply:
That's a semantic change as well, adding an additional null check.  But as this is only for debugger display, seems reasonable.

Reply:
Ah right so using ?? Transforms into an extra null check. Good to know :smile:

Reply:
More specifically, the existing code is the equivalent of:
```C#
Delegate d= m_action;
if (d != null)
{
    return d.Method.ToString();
}
return "{null}";
```
The version you proposed is the equivalent of:
```C#
Delegate d = m_action;
if (d != null)
{
    string s = d.Method.ToString();
    if (s != null)
    {
        return s;
    }
}
return "{null}";
```